docs/docs/dependency-specification.md

```diff
@@ -139,6 +139,23 @@ with the corresponding `add` call:
 poetry add https://example.com/my-package-0.1.0.tar.gz
 ```
 
+## `source` dependencies
+
+To depend on a package from an [alternate repository](/docs/repositories/#install-dependencies-from-a-private-repository),
+you can use the `source` property:
+
+```toml
+[[tool.poetry.source]]
+name = "foo"
+url = "https://foo.bar/simple/"
+secondary = true
+
+[tool.poetry.dependencies]
+my-cool-package = { version = "*", source = "foo" }
+```
+
+!!!note
+    In this example, we expect `foo` to be configured conrrectly. See [using a private repository](repositories.md#using-a-private-repository) for futher information.
```

Comment:
Found a typo XD (there was an `n` in `correctly`)

Also added a newline to be similar to the other notes here in this document.

```suggestion
!!!note

    In this example, we expect `foo` to be configured correctly. See [using a private repository](repositories.md#using-a-private-repository) for futher information.
```

Reply:
it's really hard to find that. XD